crates/frontend/src/components/context_form.rs

```diff
@@ -41,16 +46,17 @@ where
         handle_change(f_context.clone());
     });
 
-    let handle_select_dropdown_option = move |selected_dimension: Dimension| {
-        let dimension_name = selected_dimension.dimension;
-        set_context.update(|value| {
-            logging::log!("{:?}", value);
-            value.push((dimension_name.to_string(), "".to_string(), "".to_string()))
+    let handle_select_dropdown_option =
+        Callback::new(move |selected_dimension: Dimension| {
+            let dimension_name = selected_dimension.dimension;
+            set_context.update(|value| {
+                logging::log!("{:?}", value);
```

Comment:
Can we add a reference string to indicate what is being logged? If this is not needed we can remove it.